integration/e2e/service.go

```diff
@@ -394,12 +411,71 @@ func (s *HTTPService) HTTPEndpoint() string {
 	return s.Endpoint(s.httpPort)
 }
 
-func (s *HTTPService) NetworkHTTPEndpoint(networkName string) string {
-	return s.NetworkEndpoint(networkName, s.httpPort)
+func (s *HTTPService) NetworkHTTPEndpoint() string {
+	return s.NetworkEndpoint(s.httpPort)
+}
+
+func (s *HTTPService) NetworkHTTPEndpointFor(networkName string) string {
+	return s.NetworkEndpointFor(networkName, s.httpPort)
+}
+
+func sumValues(family *io_prometheus_client.MetricFamily) float64 {
+	sum := 0.0
+	for _, m := range family.Metric {
+		if m.GetGauge() != nil {
+			sum += m.GetGauge().GetValue()
+		} else if m.GetCounter() != nil {
+			sum += m.GetCounter().GetValue()
+		} else if m.GetHistogram() != nil {
+			sum += m.GetHistogram().GetSampleSum()
```

Comment:
Before we were using `GetSampleCount()`. This is the reason why integration tests are breaking. The `GetSampleSum()` is more correct and I'm OK keeping it.

To fix the tests, an option is to replace the `e2e.Equals()` on the metric `cortex_dynamo_sync_tables_seconds` with a `e2e.Greater(0)` (adding `Greater()` should be straightforward).

Reply:
:+1: